Cargo.toml

```diff
@@ -20,7 +20,7 @@ name = "alink"
 
 [dependencies]
 amplify = "4.0.0"
-aluvm = { version = "0.10.2", features = ["std", "secp256k1"] }
+aluvm = { git = "https://github.com/6293/rust-aluvm", branch = "bytes-put", features = ["std", "secp256k1"] }
```

Comment:
Can you pls change that to `patch.crates-io` section and remove the branch name (since the PR was merged)?

Reply:
done